airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/ConnectionName.tsx

```diff
@@ -53,7 +47,10 @@ const ConnectionName: React.FC<ConnectionNameProps> = ({ connection }) => {
     try {
       setLoading(true);
 
-      await updateConnection(buildConnectionUpdate(connection, { name: connectionNameTrimmed }));
```

Comment:
What happened to the `buildConnectionUpdate` functionality?

Reply:
With the addition of PATCH-style logic it's no longer necessary!